lib/index.js

```diff
@@ -1,39 +1,49 @@
+'use strict';
+
 var AWS = require('aws-sdk');
-AWS.config.region = process.env.AWS_REGION;
```

Comment:
Do we no longer _need_ to configure the `AWS_REGION` before instantiating the `SES` client? ❓